### PR TITLE
Adding my Barraquito recipe

### PR DESCRIPTION
**Pull request by RichardVerbraak**
My first ever pull request! Hoping I got it right!

### CODE REVIEW

**Comment by RichardVerbraak** [comment]
Lovely spanish coffee recipe!

File: src/recipes/barraquito.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "Frothed Milk",
+        "Condensed Milk"
+    ],
+    "directions": [
+        "Step 1. Add the condensed milk to the coffee, followed by the licor.",
+        "Step 2. Add the espresso and optional lemon peel.",
+        "Step 3. Add the frothed milk and sprinkle some cinammon for some extra flavor."
+    ],
+    "image": "barraquito.jpg",
+    "source": "<url if available>",
```
Comment (alfg) [inline src/recipes/barraquito.json]: This line can be removed if no url source is provided.
```suggestion
```

File: src/recipes/barraquito.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "keywords": [
+        "Coffee",
+        "Spanish",
+        "Canary Isles"
```
Comment (alfg) [inline src/recipes/barraquito.json]: Can you make these lowercase? Thanks!

Comment (RichardVerbraak) [reply src/recipes/barraquito.json]: Also done! I'm not sure how to update a PR but I added an updated file just now.

**Comment by alfg** [comment]
Thanks. Sorry for the delayed response.

**Pull request merged**